README.md

```diff
@@ -175,6 +175,10 @@ Due to their public nature, GitHub and mailing lists are NOT appropriate places
 
 [disclosure]: https://coreos.com/security/disclosure/
 
+## etcd Emeritus
```

Comment:
etcd Emeritus Maintainers

I would also add an explanation. Something along the lines of:

"These emeritus maintainers dedicated a part of their career to etcd and reviewed code, triaged bugs, and pushed the project forward over a substantial period of time. Thank you."

Reply:
updated PTAL cc/ @philips